tests/tests_integration/test_api/test_data_modeling/test_containers.py

```diff
@@ -101,4 +98,53 @@ def test_retrieve_non_existent(self, cognite_client: CogniteClient):
 
     def test_iterate_over_containers(self, cognite_client: CogniteClient):
         for containers in cognite_client.data_modeling.containers(chunk_size=2, limit=-1):
-            assert isinstance(containers, models.ContainerList)
+            assert isinstance(containers, dm.ContainerList)
+
+    def test_apply_invalid_container(self, cognite_client: CogniteClient):
+        with pytest.raises(CogniteAPIError) as error:
+            cognite_client.data_modeling.containers.apply(
+                dm.ContainerApply(
+                    space="nonExistingSpace",
+                    external_id="myContainer",
+                    properties={"name": dm.ContainerProperty(type=dm.Text())},
+                    used_for="node",
+                )
+            )
+
+        # Assert
+        assert error.value.code == 400
+        assert "One or more spaces do not exist" in error.value.message
+
+    def test_apply_failed_and_successful_task(
```

Comment:
Let's make sure we have these tests for all the resource types.

Reply:
Working on it :)